### PR TITLE
[zh-cn]: create the translation of `Permission-Policy.ambient-light-sensor` directive

### PR DESCRIPTION
**Pull request by tianyeeT**
…ensor` directive

### CODE REVIEW

File: files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Permissions-Policy: ambient-light-sensor"
+slug: Web/HTTP/Headers/Permissions-Policy/ambient-light-sensor
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
+---
+
+{{HTTPSidebar}} {{SeeCompatTable}}
+
+HTTP {{HTTPHeader('Permissions-Policy')}} 头部的 `ambient-light-sensor` 指令控制当前文档是否被允许通过 {{domxref('AmbientLightSensor')}} 接口收集设备周围环境中的光照信息。
```
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md:10]:
```suggestion
HTTP {{HTTPHeader('Permissions-Policy')}} 标头的 `ambient-light-sensor` 指令控制当前文档是否被允许通过 {{domxref('AmbientLightSensor')}} 接口收集设备周围环境中的光照信息。
```

File: files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{HTTPHeader('Permissions-Policy')}} 头部
```
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md:37]:
```suggestion
- {{HTTPHeader('Permissions-Policy')}} 标头
```

File: files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+具体来说，如果定义的策略阻止使用此功能，则 {{domxref("AmbientLightSensor.AmbientLightSensor", "AmbientLightSensor()")}} 构造函数调用将抛出类型为 `SecurityError` 的 {{domxref("DOMException")}}。
+
+## 语法
+
+```http
+Permissions-Policy: ambient-light-sensor=<allowlist>;
+```
+
+- `<allowlist>`
+  - : 允许使用该功能的来源列表。请参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#syntax) 了解更多详细信息。
```
Comment (PassionPenguin) [inline files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md:21]:
```suggestion
  - : 允许使用该功能的来源列表。请参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法) 了解更多详细信息。
```

**Review by yin1999 — COMMENTED** [review]
https://github.com/mdn/translated-content/pull/22051#pullrequestreview-2147211702

File: files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Permissions-Policy: ambient-light-sensor"
```
Comment (skyclouds2001) [inline files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md:2]: 同 https://github.com/mdn/translated-content/pull/22049#discussion_r1661735586

File: files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+slug: Web/HTTP/Headers/Permissions-Policy/ambient-light-sensor
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
+---
+
+{{HTTPSidebar}} {{SeeCompatTable}}
+
+HTTP {{HTTPHeader('Permissions-Policy')}} 标头的 `ambient-light-sensor` 指令控制当前文档是否被允许通过 {{domxref('AmbientLightSensor')}} 接口收集设备周围环境中的光照信息。
+
+具体来说，如果定义的策略阻止使用此功能，则 {{domxref("AmbientLightSensor.AmbientLightSensor", "AmbientLightSensor()")}} 构造函数调用将抛出类型为 `SecurityError` 的 {{domxref("DOMException")}}。
```
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md:12]:
```suggestion
具体来说，如果定义的策略阻止使用此特性，则 {{domxref("AmbientLightSensor.AmbientLightSensor", "AmbientLightSensor()")}} 构造函数调用将抛出类型为 `SecurityError` 的 {{domxref("DOMException")}}。
```

File: files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Permissions-Policy: ambient-light-sensor"
+slug: Web/HTTP/Headers/Permissions-Policy/ambient-light-sensor
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
+---
+
+{{HTTPSidebar}} {{SeeCompatTable}}
+
+HTTP {{HTTPHeader('Permissions-Policy')}} 标头的 `ambient-light-sensor` 指令控制当前文档是否被允许通过 {{domxref('AmbientLightSensor')}} 接口收集设备周围环境中的光照信息。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/ambient-light-sensor/index.md:10]:
```suggestion
HTTP {{HTTPHeader('Permissions-Policy')}} 标头的 `ambient-light-sensor` 指令控制当前文档是否被允许通过 {{domxref('AmbientLightSensor')}} 接口收集设备周围环境中的光亮信息。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**